test-app/tests/acceptance/routeable-engine-demo-refresh-test.js

```diff
@@ -0,0 +1,43 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { visit, find, click } from '@ember/test-helpers';
+
+module('Acceptance | Engine Router Service | Refresh Method', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('refresh without params triggers refresh with current route', async function (assert) {
+    await visit('/routable-engine-demo/ember-blog/new');
+
+    let counter = await find('.route-refresh-counter').textContent;
```

Comment:
```suggestion
    let counter = find('.route-refresh-counter').textContent;
```
super extra minor, this is sync operation